measure.rb

```diff
@@ -4585,13 +4590,9 @@ def self.check_surface_assembly_rvalue(runner, surfaces, film_r, assembly_r, mat
 
       if surface.adjacentFoundation.is_initialized
         foundation = surface.adjacentFoundation.get
-        if foundation.interiorVerticalInsulationMaterial.is_initialized
-          int_mat = foundation.interiorVerticalInsulationMaterial.get.to_StandardOpaqueMaterial.get
-          constr_r += UnitConversions.convert(int_mat.thickness, "m", "ft") / UnitConversions.convert(int_mat.thermalConductivity, "W/(m*K)", "Btu/(hr*ft*R)")
-        end
-        if foundation.exteriorVerticalInsulationMaterial.is_initialized
-          ext_mat = foundation.exteriorVerticalInsulationMaterial.get.to_StandardOpaqueMaterial.get
-          constr_r += UnitConversions.convert(ext_mat.thickness, "m", "ft") / UnitConversions.convert(ext_mat.thermalConductivity, "W/(m*K)", "Btu/(hr*ft*R)")
+        foundation.customBlocks.each do |custom_block|
+          ins_mat = custom_block.material.to_StandardOpaqueMaterial.get
+          constr_r += UnitConversions.convert(ins_mat.thickness, "m", "ft") / UnitConversions.convert(ins_mat.thermalConductivity, "W/(m*K)", "Btu/(hr*ft*R)")
```

Comment:
I removed the codes of getting interior/exterior vertical insulation r values here because foundation walls have switched to using custom blocks. Sometimes, slab will use interior/exterior vertical insulation too, but it never calls this function. Is it ok to remove this or is it better to keep it for future use of interior/exterior vertical insulation inputs?

Reply:
Always good to remove unused code -- it's available in git if we ever need it again in the future. Good change.